src/ppl/src/IOPlacer.cpp

```diff
@@ -205,18 +204,20 @@ void IOPlacer::randomPlacement(std::vector<int> pin_indices, std::vector<int> sl
 
   std::vector<Slot> &slots = top_layer ? top_layer_slots_ : slots_;
 
+  std::vector<IOPin>& io_pins = netlist_.getIOPins();
+  int io_idx = 0;
   for (int pin_idx : pin_indices) {
-    int b = vIOs[0];
+    int b = vIOs[io_idx];
```

Comment:
Could we give this a more descriptive variable name?

Reply:
Thanks for the comment! I've changed the var name.